tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__;
+        });
+    }
+
+    private function getMix()
+    {
+        return new \Illuminate\View\Mix\Mix;
```

Comment:
No need to declare a useless method here.
`$this->mix = new \Illuminate\View\Mix\Mix;` can be in `setUp` method.